developer/pages-router/installation.mdx

```diff
@@ -150,7 +150,7 @@ Create an [optional catch-all route](https://nextjs.org/docs/pages/building-your
 
 **Important notes**:
 
-1. If you already have an `index.tsx` file, you will need to name the file `[...path].tsx` instead of `[[...path]].tsx`. For more information about the differences between catch-all and optional catch-all segments, refer to the Next.js [Catch-all segments](https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes#catch-all-segments) documentation.
+1. If you already have a root page file (`index.tsx`), delete it to ensure that all pages (including the home page) are managed by Makeswift.
```

Comment:
I think we should keep the instructions for the case where not all pages are managed by Makeswift. They enable incremental adoption, and it's an important use case in its own right.

Reply:
Yeah that's an interesting one. We recommend replacing in app router installation. I would think if you're using Makeswift, you want it managing all of your pages? Maybe that't not true? I can revert back and add a tiny bit to help clarify what happens

Reply:
Updated!